mne/preprocessing/ica.py

```diff
@@ -254,7 +254,8 @@ def __repr__(self):
               hasattr(self, 'n_components_') else
               'no dimension reduction')
         if self.info is not None:
-            ch_fit = ['"%s"' % c for c in ['mag', 'grad', 'eeg'] if c in self]
+            ch_fit = ['"%s"' % c for c in ['mag', 'grad', 'eeg', 'seeg']
```

Comment:
wherever we put the `_pick_data_channels`, we should add a `_DATA_TYPES = ('mag', 'grad', 'eeg', 'seeg')` because this comes up a lot (even twice in this PR)